content/api/commands/mount.md

```diff
@@ -49,7 +49,33 @@ Cypress.Commands.overwrite('mount', (jsx, options) => {
 ```
 
 </template>
-<template #vue>
+<template #vue2>
+
+```js
+import { mount } from '@cypress/vue'
+
+Cypress.Commands.overwrite('mount', (comp, options = {}) => {
+  // Setup options object
+  options.extensions = options.extensions || {}
+  options.extensions.plugins = options.extensions.plugins || []
+  options.extensions.components = options.extensions.components || {}
```

Comment:
I feel like this kind of thing is better written like:

```js
  if (!options.extensions) {
    options.extensions = {}
  }
  if (!options.extensions.plugins) {
    options.extensions.plugins = []
  }
  if (!options.extensions.components) {
    options.extensions.components = {}
  }
```

Which is especially true in the "Vue Router" example, where we have:

```js
  options.router =
    options.router ||
    createRouter({
      routes: routes,
      history: createMemoryHistory(),
    })
```

Which could be:

```js
  if (!options.router) {
    options.router = createRouter({
      routes: routes,
      history: createMemoryHistory(),
    })
  }
```

Reply:
i agree with the vue router sample, but in the other ones it triples the lines of code to wrap them in if statements, and some samples have more options set up to do than others. Im cautious of making the commands too long. Thoughts?